src/examples/learn_quant/scripts/generation_test.py

```diff
@@ -0,0 +1,13 @@
+from yaml import dump
+
+try:
+    from yaml import CDumper as Dumper
+except ImportError:
+    from yaml import Dumper
+
+from examples.learn_quant.grammar import quantifiers_grammar
+from examples.learn_quant.meaning import quantifiers_universe
+
+if __name__ == "__main__":
```

Comment:
I'd say before full merge and everything, this file can be deleted :).  If you want / it's necessary, can also have tests for individual components in thsi example, separately from the main ultk tests